pkg/kv/txn_coord_sender.go

```diff
@@ -496,6 +496,11 @@ func (tc *TxnCoordSender) OnFinish(onFinishFn func(error)) {
 	tc.mu.onFinishFn = onFinishFn
 }
 
+// DisablePipelining is part of the client.TxnSender interface.
```

Comment:
Are there concerns with data races here when the txn is used concurrently? Maybe just something to comment on.

Reply:
Oh, looks like this is protected by a Mutex in the parent. Nevermind, I think.

Reply:
And it'll error out when not the first thing in the txn. Ok, we're probably good... :-)